### ISSUE TITLE
[MU4 Task] [Accessibility] New Score Wizard: Sсreen reader doesn't read some elements

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#9217]
**To Reproduce**

**Screenshots**

![02](https://user-images.githubusercontent.com/90187801/134217123-6c3ed15d-2fee-4be9-80da-941f932d6086.PNG)

**Desktop:**
- Narrator on Win10  
- Orca on Linux Mint 20.2
